content/advanced/330_servicemesh_using_appmesh/appmesh_installation/install_appmesh.md

```diff
@@ -0,0 +1,166 @@
+{{< /output >}}
+
+
+#### Create an IAM Role for the app mesh controller service account
+
+Create App Mesh Namespace for the controller
+```bash
+kubectl create ns appmesh-system
+```
+{{< output >}}
+namespace/appmesh-system created
+{{< /output >}}
+
+ Create the Service Account for App Mesh controller in the namespace we just craeted. Attach IAM Policies for AWS App Mesh and AWS Cloud Map full access. This will create identity provider IAM Role for Service Accounts so that EKS cluster and its nodes have the ability to work with the app mesh service on your behalf.
+
+{{% notice tip %}}
+if you are new to the `IAM Roles for Service Accounts (IRSA)` concept, [Click here](/beginner/110_irsa/) for me information.
+{{% /notice %}}
+
+```bash
+eksctl create iamserviceaccount \
+    --cluster eksworkshop-eksctl \
+    --namespace appmesh-system \
+    --name appmesh-controller \
+    --attach-policy-arn  arn:aws:iam::aws:policy/AWSCloudMapFullAccess,arn:aws:iam::aws:policy/AWSAppMeshFullAccess \
+    --override-existing-serviceaccounts \
+    --approve  
+ ```
+{{< output >}}
+[ℹ]  eksctl version 0.30.0     
+[ℹ]  using region us-west-2     
+[ℹ]  2 existing iamserviceaccount(s) (kube-system/aws-node,prodcatalog-ns/prodcatalog-sa) will be excluded    
+[ℹ]  1 iamserviceaccount (appmesh-system/appmesh-controller) was included (based on the include/exclude rules)     
+[ℹ]  1 iamserviceaccount (kube-system/aws-node) was excluded (based on the include/exclude rules)     
+[!]  metadata of serviceaccounts that exist in Kubernetes will be updated, as --override-existing-serviceaccounts was set    
+[ℹ]  1 task: { 2 sequential sub-tasks: { create IAM role for serviceaccount "appmesh-system/appmesh-controller", create serviceaccount "appmesh-system/appmesh-controller" } }     
+[ℹ]  building iamserviceaccount stack "eksctl-eksworkshop-eksctl-addon-iamserviceaccount-appmesh-system-appmesh-controller"     
+[ℹ]  deploying stack "eksctl-eksworkshop-eksctl-addon-iamserviceaccount-appmesh-system-appmesh-controller"  
+[ℹ]  created serviceaccount "appmesh-system/appmesh-controller 
+{{< /output >}}
+
+#### Install App Mesh Controller
+
+This will install App Mesh Controller into the appmesh-system namespace with X-Ray enabled
+
+```bash
+helm upgrade -i appmesh-controller eks/appmesh-controller \
+    --namespace appmesh-system \
+    --set region=$AWS_REGION \
+    --set serviceAccount.create=false \
+    --set serviceAccount.name=appmesh-controller \
+    --set tracing.enabled=true \
+    --set tracing.provider=x-ray
+```
+{{< output >}}
+Release "appmesh-controller" has been upgraded. Happy Helming!     
+NAME: appmesh-controller     
+LAST DEPLOYED: Wed Jan 20 21:07:01 2021     
+NAMESPACE: appmesh-system     
+STATUS: deployed     
+REVISION: 1    
+TEST SUITE: None     
+NOTES:     
+AWS App Mesh controller installed! 
+{{< /output >}}
+
+Confirm that the controller version is v1.0.0 or later.
```

Comment:
For the version check, you may want to update output (current release is v1.3.0)

Reply:
updated